test/fixtures/e2e/bento/base-carousel.html

```diff
@@ -3,17 +3,11 @@
   <head>
     <script src="https://cdn.ampproject.org/bento.js"></script>
     <!-- These styles prevent Cumulative Layout Shift on the unupgraded custom element -->
-    <style>
```

Comment:
I'm missing context here, why are these no longer necessary?

Reply:
we'd decided to recommend consumers ensure that components avoided layout shift by link to our stylesheets rather than copy/paste these styles, but these fixtures were never updated to reflect that.  (the link rel is added on line 10)